### PR TITLE
EVG-2663 minimal UI changes to make admin page work again

### PR DESCRIPTION
**Pull request by john-m-liu**

### CODE REVIEW

File: public/static/js/admin.js [hunk 1]
```diff
@@ -28,7 +28,13 @@ mciModule.controller('AdminSettingsController', ['$scope','$window', 'mciAdminRe
     var errorHandler = function(resp) {
       notificationService.pushNotification("Error saving settings: " + resp.data.error, "errorHeader");
     }
-    mciAdminRestService.saveSettings($scope.Settings, { success: successHandler, error: errorHandler });
+    // temporary workaround until the admin page has components for every part of the config
```
Comment (richardsamuels) [inline public/static/js/admin.js]: can you throw a TODO and ticket or epic number here please.

File: public/static/js/admin.js [hunk 2]
```diff
@@ -28,7 +28,13 @@ mciModule.controller('AdminSettingsController', ['$scope','$window', 'mciAdminRe
...
+    settings = {
+      banner: $scope.Settings.banner,
+      banner_theme: $scope.Settings.banner_theme,
+      service_flags: $scope.Settings.service_flags,
```
Comment (richardsamuels) [inline public/static/js/admin.js]: won't that trailing comma break things?

File: public/static/js/admin.js (hunk 1 above)
Comment (john-m-liu) [reply public/static/js/admin.js]: done

File: public/static/js/admin.js (hunk 2 above)
Comment (john-m-liu) [reply public/static/js/admin.js]: Javascript allows trailing commas, JSON does not. I removed it though

**Pull request merged**